### PR TITLE
orders (fix) empty address city issue

### PR DESCRIPTION
**Pull request by ElGreetly**

### CODE REVIEW

**Review by HazemKhaled — CHANGES_REQUESTED** [review]

File: includes/class-dropshipping-wc-mp-orders.php [hunk 1]
```diff
@@ -269,6 +269,12 @@ public function knawat_format_order( $order_id, $is_update = false, $json = true
 				}
 			}
 		}
+
+		// If city is not available add state value
+		if (empty($new_order['shipping']['city'])) {
+            $new_order['shipping']['city'] = empty($new_order['shipping']['state']) ? $new_order['billing']['state'] : $new_order['shipping']['state'];
```
[the remark this answers is not in the document]
Comment (ElGreetly) [reply includes/class-dropshipping-wc-mp-orders.php:275]: Some themes just use the billing address for both and we already doing that here https://github.com/Knawat/dropshipping-woocommerce/pull/134/files#diff-889bca98e7d770cdc1bfe2b8d3aa66c5R263

[the remark this answers is not in the document]
Comment (HazemKhaled) [reply includes/class-dropshipping-wc-mp-orders.php:275]: In this line 100% you have shipping, so no need to get it from billing

[the remark this answers is not in the document]
Comment (ElGreetly) [reply includes/class-dropshipping-wc-mp-orders.php:275]: There are some stores example that ignores shipping address, billing address here is also client address, not the store owner

[the remark this answers is not in the document]
Comment (ElGreetly) [reply includes/class-dropshipping-wc-mp-orders.php:275]: Some stores use billing for both billing and shipping

[the remark this answers is not in the document]
Comment (ElGreetly) [reply includes/class-dropshipping-wc-mp-orders.php]: You are right ya rayyys

**Review by HazemKhaled — APPROVED** [review]

**Pull request merged**